pkg/serverapiclient/api_workspace.go

```diff
@@ -16,9 +16,11 @@ import (
 	"io"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
-)
 
+	"github.com/daytonaio/daytona/pkg/types"
```

Comment:
The `serverapiclient` package is autogenerated from OpenAPI annotations in the server API.

To generate them, use the `./hack/swagger.sh` script.

This file should not be edited manually.

Reply:
got it, generated :white_check_mark: